### ISSUE TITLE
Error while installing GraphSpace using docker image

### ISSUE THREAD

**Issue by adbharadwaj** [Murali-group/GraphSpace#349]
@melvin15may One of the undergraduate students (@heidict) in our lab is having issues installing GraphSpace using docker. Can you help? Here is the error she is getting while following the instructions given [here](https://github.com/Murali-group/GraphSpace/wiki/Setup-and-run-graphspace-docker-image).

```sh
GRAPHSPACEXDOCKER | bash: ./docker_command.sh: No such file or directory
GRAPHSPACEXDOCKER exited with code 127
```

**Comment by adbharadwaj** [comment]
We get the same error when we try the following [instructions](https://github.com/Murali-group/GraphSpace/wiki/Build-and-run-GraphSpace-docker-image-using-DockerFile).

**Comment by melvin15may** [comment]
Can you check if you have `docker_command.sh` file in the GraphSpace root directory.

**Comment by melvin15may** [comment]
The branch containing the file `docker_command.sh` has not been merged.

**Comment by adbharadwaj** [comment]
Yeah, it is there. I also updated the docker file to point to your fork (e#264 branch) instead of master repo.

**Comment by melvin15may** [comment]
Point it to `e#264-docker-diff`

**Comment by adbharadwaj** [comment]
I updated the Dockerfile [here](https://github.com/melvin15may/GraphSpace/blob/e%23264/Dockerfile#L51) to 

```sh
git clone -b e#264 https://github.com/melvin15may/GraphSpace.git && \
```

**Comment by melvin15may** [comment]
Just update the branch, should work

**Comment by adbharadwaj** [comment]
@melvin15may whats wrong with `e#264-docker-diff`? I thought that was the final code.

**Comment by melvin15may** [comment]
Sorry about that, just went through e#264 branch. It should work. Can you paste the error message.

**Comment by adbharadwaj** [comment]
```
Recreating GRAPHSPACEXDOCKER ... done
Attaching to GRAPHSPACEXDOCKER
GRAPHSPACEXDOCKER | bash: /GraphSpace/docker_config/docker_command.sh: No such file or directory
GRAPHSPACEXDOCKER exited with code 127
```

**Comment by adbharadwaj** [comment]
@melvin15may @heidict The solution was to clear the old images, containers, and volumes.

Refer to following article on how to do it.

https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes
